### ISSUE TITLE
Upsert() is not an function

### ISSUE THREAD

**Issue by RustedRad** [discordjs/guide#452]
if u follow the discordjs.guide tutorial we can see that there is an  error that says upsert is not and function
here is the error
PS C:\Users\user\Desktop\bot-tutorial-season-2-master> node dbInit.js
TypeError: CurrencyShop.upsert is not a function

**Comment by almostSouji** [comment]
I can not reproduce this issue with sequelize `6.3.4`, please make sure you install a recent version for sequelize and follow the tutorial correctly.

**Comment by RustedRad** [comment]
Siuji I am using the version  6.3.4

**Comment by almostSouji** [comment]
I tested that guide yesterday when I applied the "import is not a function" fix and can not find any issues with it. As per the sequelize documentation native upsert has only been available for postgres backends in the past, however has been widened so that now every backend has a native upsert available through the sequelize interface.

I ran the bot, used all commands, upsert was definitely called in the process, no issues.
